.travis.yml

```diff
@@ -9,6 +9,7 @@ node_js:
   - '10'
 script:
   - npm run bundle
+  - npm run bundle-min
```

Comment:
Yep, afaik we deploy both sets of files. An improvement would be to build once and minify the built files rather than doing this, but it would require a bit of a rollup config change that I didn't want to make for this PR